### DISCUSSION TITLE
Issue with Excluding Specific Recurring Events with a time slot (Start and End Times)

### DISCUSSION THREAD

**Discussion by Infinity-chakra** [YSHgroup/vue3-fullcalendar-bootstrap#2] [Q&A] [answered]
Hi everyone,

I am encountering an issue when attempting to exclude specific recurring events with start and end times using the `exrule` property in FullCalendar library. Despite following the [recommended approach](https://fullcalendar.io/docs/rrule-plugin), the exclusions are not being applied correctly.

Here, reproduced my case.

```
// example dates to excluded from the recurring events
excluded_dates: [ "2024-03-08",  "2024-03-15",  "2024-03-08"]
```

```TypeScript
events: [{
  rrule: ...
  ...
  exdate: !isSpecificEntry ? ('excluded_dates' in entry ? entry.excluded_dates : []) : [],
}]
```

But it works only in case if the event is the full day.
So, I tried again with following and expected to result the recurring events with start and end times should be correctly excluded based on the provided `exrule` objects.

```TypeScript
events: [{
  rrule: ...
  ...
  // I'd also tried only with the `exrule` alone.
  exdate: !isSpecificEntry ? entry.excluded_dates : [],
  exrule: !isSpecificEntry ?  entry.excluded_dates.map(genExrule): [],
}]
...
const genExrule = (excluded_date: string) => {
  const startDateTime = moment(excluded_date).startOf('day').format('YYYY-MM-DD HH:mm');
  const endDateTime = moment(excluded_date).endOf('day').format('YYYY-MM-DD HH:mm');
  
  return {
    freq: 'weekly',
    dtstart: startDateTime,
    until: endDateTime
  };
};
```

But the specified events are not being excluded as expected. The calendar continues to display the recurring events on the excluded dates.

Please let me know if any further information or code samples are needed to investigate and resolve this issue.

**Comment by MadSmiley** [comment, marked as answer]
Put the `exdate` in the same format as your `dtstart` 
For the time, just take the same as your `dtstart`

**Reply by Infinity-chakra** [reply]
Thank you for your answer.
For now, I believe the approach is the best solution.
If we can find the updates in the fullcalendar library, please let me know.
Thank you once more.
